[3.9] ansible-service-broker fails to deploy due to missing namespace configuration

#### Description

When deploying ASB on Origin 3.9, it fails due to missing namespace configuration.

##### Version

```
openshift-ansible-3.9.23-1
```

##### Steps To Reproduce
1. deploy service catalog with glusterfs (storage not likely an issue here)
2. check logs of container created by asb-1-deploy


##### Expected Results

Deployment of ASB

##### Observed Results

ASB fails to deploy fully.

```
$ oc -n openshift-ansible-service-broker logs po/asb-1-<random>

Using config file mounted to /etc/ansible-service-broker/config.yaml
============================================================
==           Starting Ansible Service Broker...           ==
============================================================
[2018-04-19T18:52:40.727Z] [NOTICE] - Initializing clients...
2018/04/19 18:52:40 Unable to get log.logfile from config
[2018-04-19T18:52:40.728Z] [INFO] - == ETCD CX ==
[2018-04-19T18:52:40.728Z] [INFO] - EtcdHost: asb-etcd.openshift-ansible-service-broker.svc
[2018-04-19T18:52:40.728Z] [INFO] - EtcdPort: 2379
[2018-04-19T18:52:40.728Z] [INFO] - Endpoints: [https://asb-etcd.openshift-ansible-service-broker.svc:2379]
[2018-04-19T18:52:40.745Z] [INFO] - Etcd Version [Server: 3.3.3, Cluster: 3.3.0]
[2018-04-19T18:52:40.754Z] [INFO] - OpenShift version: v3.9.0+8e3cb9b-8
[2018-04-19T18:52:40.769Z] [INFO] - Kubernetes version: v1.9.1+a0ce1bc657
[2018-04-19T18:52:40.776Z] [ERROR] - Unable to load secret 'asb-registry-auth' from namespace ''
[2018-04-19T18:52:40.776Z] [ERROR] - Unable to retrieve registry auth: Failed to find Dockerhub credentials in secret: asb-registry-auth
[2018-04-19T18:52:40.776Z] [ERROR] - Failed to initialize dh Registry err - Failed to find Dockerhub credentials in secret: asb-registry-auth 
```

##### Additional Information

Deployed from openshift-ansible, release-3.9 branch on CentOS Atomic. Note the line where `asb-registry-auth` is being pulled from namespace `''`.

Change here may be necessary for 3.9?

https://github.com/openshift/openshift-ansible/commit/e498e528bbe3359c5df7e47a4ea0ab4532045063

--- a/roles/ansible_service_broker/tasks/install.yml
+++ b/roles/ansible_service_broker/tasks/install.yml
@@ -417,6 +417,7 @@
               host: ""
               ca_file: ""
               bearer_token_file: ""
+              namespace: openshift-ansible-service-broker
               sandbox_role: {{ ansible_service_broker_sandbox_role }}
               image_pull_policy: {{ ansible_service_broker_image_pull_policy }}
               keep_namespace: {{ ansible_service_broker_keep_namespace | bool | lower }}

